File: centre-ui/src/app/services/models/student-details-response.ts
/* tslint:disable */
/* eslint-disable */
import { AddressDto } from '../models/address-dto';
import { ParentDto } from '../models/parent-dto';
import { ParentStatusDto } from '../models/parent-status-dto';
export interface StudentDetailsResponse {
  address?: AddressDto;
  email?: string;
  father?: ParentDto;
  firstName?: string;
  id?: number;
  lastName?: string;
  levelOfEducation?: string;
  mother?: ParentDto;
  parentStatus?: ParentStatusDto;
  phoneNumber?: string;
}

File: centre-api/src/main/java/com/centre/poly/person/dto/StudentDetailsResponse.java
package com.centre.poly.person.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class StudentDetailsResponse {
    Long id;
    private String firstName;
    private String lastName;
    private String phoneNumber;
    private String email;
    private String levelOfEducation;
    
    private AddressDTO address;
    private ParentDTO mother;
    private ParentDTO father;
    
    private ParentStatusDTO parentStatus;
    
    
}
